.gitlab/package_build/installer.yml

```diff
@@ -104,10 +104,14 @@ installer-install-scripts:
     - VERSION="$(inv agent.version --url-safe)-1" || exit $?
     - echo "About to build for $VERSION"
     - mkdir -p $OMNIBUS_PACKAGE_DIR
-    - inv -e installer.build-linux-script "default" "$VERSION"
-    - inv -e installer.build-linux-script "databricks" "$VERSION"
-    - inv -e installer.build-linux-script "emr" "$VERSION"
-    - inv -e installer.build-linux-script "dataproc" "$VERSION"
+    - ls -la $OMNIBUS_PACKAGE_DIR
+    - tar -tvJf $OMNIBUS_PACKAGE_DIR/datadog-installer-7*amd64.tar.xz
```

Comment:
This only lists the archive content, right? If so, I suppose it can be removed (but I don't mind it staying either)

Reply:
good catch, it was some leftover debugging